src/_includes/layouts/base.njk

```diff
@@ -31,6 +31,10 @@
     <div class="content-wrapper">
       <main id="mc" class="container-xl mx-auto xl:px-0 sm:px-500 px-300 general-layout">
         {% block content %}{{ content | safe }}{% endblock %}
+
+        <gcds-date-modified class="mb-400 font-size-caption">
```

Comment:
Can you change `font-size-caption` to `font-caption` so that it uses the caption line height as well?